### ISSUE TITLE
Choice of Javascript Graphics Library for the website

### ISSUE THREAD

**Issue by EmilZach** [EmilZach/Sentence-Crusher#17]
This is more about planning of things, since we are moving to front-end development (the look and functionality of a website).

I've looked into several graphics libraries (some in this link in the category general-purpose: [https://github.com/mathjax/MathJax-docs/wiki/List-of-general-purpose-graphics-javascript-libraries]), especially a library called BonsaiJS. I am currently looking at the documentation for BonsaiJS, and I can say that it looks easy to use and since we are not going to make anything advanced, I would personally consider this library.

However, since we are working as a team, I am asking everyone to look at the libraries in the link and the library I listed here. I want to hear your opinion so we can have a discussion about which library can be good for our website and our objective.

Good afternoon, and happy programming!

**Comment by Arxcis** [comment]
I welcome your initiative greatly. Personally, I want to try and get it to work at all in the browser, before exploring what nice libraries can be but on top :)

**Comment by Arxcis** [comment]
I welcome your initiative greatly. Personally, I want to try and get it to work at all in the browser, before exploring what nice libraries can be but on top :)

**Comment by Arxcis** [comment]
Sorry, misscliked

**Comment by EmilZach** [comment]
Yes, we obviously have to make it work, but I thought of an idea that's better explained when we are at the course. See you all soon. :-)
